### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by danicatt**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1,18 +1,20 @@
 <?php
 require('functions.php');
+$db = getDatabase();
+$cartoonArray = queryDatabase($db);
 ?>
 
 <!DOCTYPE html>
```
Comment (alexjennings56) [inline index.php:7]: Unless mike doesn't mind, I don't think it's needed?

File: stylesheet.css [hunk 2]
```diff
@@ -5,7 +5,6 @@
 html {
     font-size: 62.5%;
     font-family: helvetica;
```
Comment (alexjennings56) [inline stylesheet.css:7]: i would put font-family in the body but if it works then i guess it doesn't matter :)

File: test/functionTest.php [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?php
+require('functions.php');
+use PHPunit\Framework\TestCase;
+
+class functionTest extends TestCase {
+
+    public function testSuccessloopCartoons() {
+        $expected = "<div class='eachCharacter'><h2 class='charName'>Test Name</h2><p class='cardContent'>From: Test Show</p><p class='cardContent'>Species: Test Species</p><p class='cardContent'>Catchphrase: Test Phrase</p><img src=Test Image></div>";
```
Comment (alexjennings56) [inline test/functionTest.php:8]: maybe split this into a few lines just so it isn't so long?
```suggestion
        $expected = "<div class='eachCharacter'><h2 class='charName'>Test Name</h2>
        <p class='cardContent'>From: Test Show</p><p class='cardContent'>Species: Test Species</p>
        <p class='cardContent'>Catchphrase: Test Phrase</p><img src=Test Image></div>";
```

**Review by alexjennings56 — APPROVED** [review]
Looks good :)

File: test/functionTest.php (hunk 3 above)
Comment (danicatt) [reply test/functionTest.php:8]: I just tried, as it for readability makes sense, but when I retested the code, it came back as a failed :(

File: stylesheet.css (hunk 2 above)
Comment (danicatt) [reply stylesheet.css:7]: good call, moved it to the body.

File: index.php (hunk 1 above)
Comment (danicatt) [reply index.php:7]: Yeah I wasn't sure if needed.

**Pull request merged**